raspisump/database.py

```diff
@@ -0,0 +1,73 @@
+""" Module to write water level reading to a database."""
+
+# Raspi-sump, a sump pump monitoring system.
+# Al Audet
+# http://www.linuxnorth.org/raspi-sump/
+# Module by: USAF_Pride
+#
+# All configuration changes should be done in raspisump.conf
+# MIT License -- http://www.linuxnorth.org/raspi-sump/license.html
+
+
+import sqlite3
+
+# here just in case
+try:
+    import ConfigParser as configparser  # Python2
+except ImportError:
+    import configparser  # Python3
+
+config = configparser.RawConfigParser()
+config.read("/home/pi/raspi-sump/raspisump.conf")
+
+configs = {
+}
+
+database_path = "/home/pi/raspi-sump/data/"
+
+
+def write_database(water_depth, databasename):
+# -----------------------------------------------------------------------------$
+#   Create our database & table if it doesn't already exist...
+
+    sqliteDB = databasepath + databasename
+# Connecting to the database file
+    conn = sqlite3.connect( sqliteDB, isolation_level=None)
+    c = conn.cursor()
+
+# Here be Dragons!!!
+# Set to "1 == 1" to drop an existing table...
+# This will remove all existing data with no warning!
+# You have been warned!!!!!!
+    if ( 1 == 0):   # <--- Dragon lies here!!!
+        sqlStmt = "'DROP TABLE IF EXISTS '" + databasename + "';'"
+        c.execute( sqlStmt )
+        conn.commit()
+
+    sqlStmt = """
+        CREATE TABLE IF NOT EXISTS " + databasename + " (
+        insDate		datetime,
```

Comment:
sqlite does not have a datetime type https://www.sqlite.org/datatype3.html. I believe this may get converted to TEXT if called through python, but would not work directly in sqlite.  Most databases store times as 64bit integers so I was surprised to see this.  Saving it as TEXT is not wrong, its easy to read but a little more effort to utilize with other applications pulling data out (like the web interface for charting that does not exist yet)